src/carousel/carousel.ts

```diff
@@ -53,6 +61,11 @@ export class NgbCarousel implements AfterContentChecked,
   @Input() interval: number = 5000;
   @Input() wrap: boolean = true;
 
+  @HostBinding('tabIndex')
```

Comment:
As per docs,
-1 won't allow you to have sequential focus navigation
0 will make browser takes care of order in sequential  focus navigation
And 1 or any other positive value for enforcing the order 

I think this is why we use tabindex and also for an element to be focusable we need to set tab index focus flag.

Yes we can achieve it with host.